options/locale/locale_en-US.ini

```
@@ -118,6 +118,20 @@ footer = Footer
 footer.software = About Software
 footer.links = Links
 
+[editor]
+header.tooltip = Add heading
+bold.tooltip = Add bold text
+italic.tooltip = Add italic text
+quote.tooltip = Quote text
+code.tooltip = Add code/highlighted text
```

```suggestion
code.tooltip = Add code
```

---

The problem I can see with that:
I think `code` is surprisingly unspecific.
Especially since it is basically just text highlighting.
When I read it myself, I have no idea what the result of clicking that button will be, thus making the tooltip useless.
What about the opposite?
```suggestion
code.tooltip = Add highlighted text
```

---

It will result in the text wrapped in `<code>`, so it seemed descriptive enough to me. Highlight seems incorrect as that sounds to me like syntax highlighting, which this isn't.

---

Don't you agree that essentially all stuff in backticks is "code"?